flexmeasures/ui/templates/assets/asset_properties.html

```diff
@@ -245,152 +258,1061 @@ <h3>All child assets for {{ asset.name }}
+                    stringTabContentSaveBtn.className = 'btn btn-secondary btn-sm me-2 mt-2';
+                    stringTabContentSaveBtn.textContent = 'Set Value';
+                    stringTabContentSaveBtn.onclick = function () {
+                        const input = document.getElementById('flexStringInput');
+                        if (!input.value) {
+                            showToast("Please enter a value", "error");
+                            return;
+                        }
+                        if (isArrayField && selectedIndex() !== null) {
+                            flexModel[name][selectedIndex()] = input.value;
+                        } else {
+                            flexModel[name] = input.value;
+                        }
+
+                        setFlexModel(flexModel);
+                        updateFlexModel();
+                    };
+
+                    const flexInputGroup = document.createElement('div');
+                    flexInputGroup.className = 'flex-input-group mb-1 p-2';
+                    const inputLabel = document.createElement('label');
+                    inputLabel.setAttribute('for', 'fixed-value');
+                    inputLabel.className = 'form-label';
+                    inputLabel.innerHTML = `Value for <b>${getFlexFieldTitle(name)} ${isArrayField ? '| Item ' + (selectedIndex() + 1) : ''}</b>
+                        <i class="fa fa-info-circle ps-2"
+                            data-bs-toggle="tooltip"
+                            data-bs-placement="bottom"
+                            title="Set a fixed value for this field. If the field is a boolean, it will be set to true or false."
+                        ></i>`;
+                    flexInputGroup.appendChild(inputLabel);
+
+                    const input = document.createElement('input');
+                    input.type = 'text';
+                    input.className = 'form-control';
+                    input.id = 'flexStringInput';
+                    input.placeholder = 'Enter a value';
+                    input.value = `${typeof value === 'string' ? value : ''}`;
+                    input.onkeydown = function (event) {
+                        if (event.key === 'Enter') {
+                            event.preventDefault();
+                            stringTabContentSaveBtn.click();
+                        }
+                    };
+                    flexInputGroup.appendChild(input);
+                    flexInputGroup.appendChild(stringTabContentSaveBtn);
+                    tabContent.appendChild(flexInputGroup);
+                } else {
+                    return [null, null];
+                }
+
+                return [tab, tabContent];
+            }
+
+            async function renderFlexInputOptions() {
+                const card = activeCard();
+                if (!card) {
+                    showToast(`Cant render input options without a selcted Field`, "info");
```

Comment:
I see this Toast just loading the properties page.

Also, I don't believe it has value for the user. They don't know what you mean with "render input options". Maybe print that to console.log and something more general to the Toast, if at all.